src/app/routes/api/v3/user/user.routes.ts

```diff
@@ -46,15 +46,15 @@ UserRouter.post('/contact/otp/verify', UserController.handleContactVerifyOtp)
 /**
  * Verify the contact verification one-time password (OTP) for the user as part
  * of the contact verification process
- * @route POST /user/whats-new/last-seen
- * @param body.latestLastSeenFeatureUpdateDate the latest seen feature update date to update user with
+ * @route POST /user/flag/last-seen
+ * @param body.date the latest seen feature update date to update user with
  * @returns 200 when user last seen feature update updates sucessfully
  * @returns 401 if user is not currently logged in
  * @returns 422 when userId does not exist in the database
  * @returns 500 when database errors occurs
  */
 UserRouter.post(
-  '/whats-new/last-seen',
+  '/flag/last-seen',
```

Comment:
suggest a more verbose endpoint like `new-features-last-seen`, since last-seen may be a little too ambiguous, like is this to track user's last login (we do track that in our schema btw hehe)?